Thêm Xóa Sửa  - Thiếu Tìm Kiếm - 1 số Bug chưa phát hiện

PRID021 opened the pull request:
Tiếp tục update ....

Commit:
Thêm Xóa Sửa  - Thiếu Tìm Kiếm - 1 số Bug chưa phát hiện

Tiếp tục sau

princ3od reviewed (commented):
- Trước khi commit nên xem một lượt toàn bộ các file => bỏ tick các thay đổi không cần thiết, các thay đổi sẽ gây ảnh hưởng đến sản phẩm đầu ra.
- Nên đặt tên các control cụ thể, không xài tiếng Việt, chú ý quy chuẩn đặt tên của cả dự án.
- Tránh đặt tên biến vô nghĩa cho những khối lệnh quan trọng, không được viết hoa chữ cái đầu tiên của tên biến (ví dụ: LopThuocKhoi).
-  Chú ý cách sử dụng BackgroundWorker, BGWK sử dụng để tạo Thread mới để chạy các lệnh có khả năng lỗi => hạn chế chương trình bị treo (freeze), crash.
- Mỗi lớp chỉ nên phục vụ một tác vụ chung nhất, ví như 3 hàm xử lí ảnh có thể gộp lại vào 1 class ImageHelper.
- Nên đọc lướt toàn bộ các phần code khác.
- Không nên bắt tay vào code tới đâu nghĩ tới đó, nên phác thảo ra giấy, trong đầu... cách code sẽ chạy rồi hẳn code để code clean hơn, hạn chế lỗi tiềm ẩn và dễ debug, gỡ lôi hơn.

princ3od committed:
clean up

princ3od committed:
Delete StudentDataAccess.cs

Commit:
Lỗi github

adasd

princ3od committed:
Revert "Lỗi github"

This reverts commit 2a037bf290597744aa87564233c4e3ea93ee688b.

princ3od committed:
Update TutteeFrame.csproj

clean up

Commit:
Chỉnh lại form

-Sợ hãi

Commit:
-d

ddd

Commit:
Thiếu chức năng tìm kiếm

Gần xong form quản lý học sinh -  thiếu làm mượt và giảm rerfesh screen khi không cần thiết

Commit:
-Chỉnh giảm refesh screen khi không cần thiết

Tiếp tục bổ sung chức năng tìm kiếm theo mã ID

Commit:
-Mặc định default khi bật tag

Commit:
--

--

Commit:
--- Xong task quản lý học sinh

Đang xem xét các bug và hạn chế xung đột tài nguyên do đa tiến trình

Commit:
Xong From Quản Lý

Commit:
-Thêm chức năng in danh sách học sinh

Commit:
Định dạng trang In(docx,exel,pdf)

Commit:
commit

Commit:
Revert "commit"

This reverts commit 2b48784e60ebebe2331808ee1672ea0628bd0623.

Commit:
Revert "Revert "commit""

This reverts commit 30c0dfc134bdff00afa5291d4be73d2568c1b6ce.

princ3od committed:
Merge branch 'master' into QuanLyHocSinh
